README.md

```diff
@@ -81,10 +82,10 @@ osio {
 
     build resources: resources
 
-    deploy resources: resources, env: 'stage'
+    deploy resources: resources, env: environment(type: 'stage')
```

Comment:
shouldn't it be `name` instead of `type` ?

Reply:
humm we can go with the name, somehow type sounds more intuitive as in which `kind` of environment?

Reply:
@hrishin is `env` an already defined method? If not how about calling it 

```groovy

deploy resources: resource, env: env('stage')
```

Reply:
@hrishin is `env` an already defined method? If not how about calling it `env` e.g.

```groovy
deploy resources: resource, env: env('stage')
```

Reply:
yes it's already defined, hence made it environment